spec/factories_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 @smurf = FactoryGirl.build(:user)
 ActiveResource::HttpMock.respond_to do |mock|
-  mock.post   "/users.json",   {"Content-Type"=>"application/json", "authorization"=>"Bearer "}, @smurf.to_json, 201, "Location" => "/users/smurf.json"
-  mock.get    "/users/smurf.json", {"Accept"=>"application/json", "authorization"=>"Bearer "}, @smurf.to_json
+  mock.post   "/users.json",   {"Content-Type" => "application/json", authorization: "Bearer "}, @smurf.to_json, 201, "Location" => "/users/smurf.json"
+  mock.get    "/users/smurf.json", {"Accept" => "application/json", authorization: "Bearer "}, @smurf.to_json
```

Comment:
Do we need to use strings here? can't we use symbols?

Reply:
We could, but the dash in "Content-Type" still needs the quotes. It would look something like this:
> {"Content-Type": "application/json", authorization: "Bearer "}

We can do that or change back to using strings as keys. I'm not sure which option is the best, but I can agree that the mixing of styles should be avoided.

Reply:
@malmers But it does look like that? I agree that we shouldn't mix the styles - so make "Authorization" a string too! 😄